server/sonar-server/src/main/java/org/sonar/server/permission/PermissionService.java

```diff
@@ -76,6 +77,11 @@ public PermissionService(DbClient dbClient, PermissionRepository permissionRepos
     return GlobalPermissions.ALL;
   }
 
+  public UserWithPermissionQueryResult findUsersWithPermission(PermissionQuery query) {
```

Comment:
It's not what have been decided when we've talked about the new architecture of WS : The XXXService layer is useless, WS should either use DAOs or a class with minimal responsability (for instance UserUpdater).
PermissionService should be split (not now) in another class to decrease its responsability and to improve the testability.